### DISCUSSION TITLE
Small GUI improvement suggestion

### DISCUSSION THREAD

**Discussion by david32** [danielhrisca/asammdf#645] [Ideas]
2 possible improvements:

1. Channel record size in Info tab changes between B/kB/MB:
![image](https://user-images.githubusercontent.com/2933880/152589839-185f8c61-1eb1-446b-9f74-38f0ced4d8bb.png)
Can be realized with a simple if statement / similar option or a library (for example https://pypi.org/project/hurry.filesize/).

2. Allow to optionally add Channel Group name in the channel list display:
![image](https://user-images.githubusercontent.com/2933880/152592362-0caff4de-5fe8-4800-8c79-e463282567f1.png)
It doesn't look good with long names, but it doesn't look well with the comment either.
same thing applied to the info tab.

Just want to start a discussion about it, perhaps over how the group naming should look in general.

**Comment by danielhrisca** [comment]
1. fixed in the ``development`` branch
2. the channel group does not have a ``name`` field this is why the comment is used. (CANape for example only displays "Channel group N")

**Comment by david32** [comment]
Thanks for 1.

2. In my demonstration (the screenshot) I used:
`channel_group.setText(0, f"Channel group {i} {group.channel_group.acq_name} ({comment})")`
or
channel_group.setText(0, f"Channel group {i} {channel_group.acq_name} ({comment})")

This of course creates a very long name in some cases, perhaps its better to shorten "Channel group" and add the group name. I don't know.

I know a lot of people just search for signal names, but it seems to me its easier when seeing the name and not channel # whatever.

Your thoughts?

**Reply by david32** [reply]
its also kind of related to #643
